### PR TITLE
Removed FLAG_NEW_SIGN_IN_MODAL + Cleanup and re-organized old comps

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6600

## Description of Changes
- Removed FLAG_NEW_SIGN_IN_MODAL
- Removed old unused login modal components
- Reorganized old reusable login modal components

## "How We Fixed It"
N/A

## Test Plan
In general ensure that AuthModal works correctly in all places. Specifically
- In no-scoped pages
- In scoped pages, test these communities (they all show different wallets) -> `/dydx`, `/cosmos`, `/solana`, `/edgeware`, `/near`, `/axie-infinity`, `/terra`, `/evmos`, `/injective`
- Test the AuthModal on mobile devices, in the above mentioned communities

## Deployment Plan
Merge after the new sign-in modal is deployed to production.

## Other Considerations
We should clean up `FLAG_NEW_SIGN_IN_MODAL` from prod env.

### CODE REVIEW

**Comment by lzach83** [comment]
Conflicts, once those are fixed i'll finish the review

File: packages/commonwealth/client/scripts/hooks/useWallets.tsx [hunk 1]
```diff
@@ -62,14 +81,14 @@ type IuseWalletProps = {
 };
 
 const useWallets = (walletProps: IuseWalletProps) => {
-  const newSignInModalEnabled = useFlag('newSignInModal');
+  const createAccountWithDefaultValues = true;
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/hooks/useWallets.tsx:84]: do we need this if this is always true

**Review by masvelio — APPROVED** [review]
so good to say goodbye :)

**Comment by mzparacha** [comment]
Requesting a re-review as the merge conflicts were pretty bad

cc: @masvelio @lzach83

**Review by Miaplacidus — APPROVED** [review]

**Pull request merged**